docs_src/design/adr/0019-Service-List.md

```diff
@@ -0,0 +1,92 @@
+  - Using Consul would make sure that the service list is well formed (meaining adherence to format, but this does not mean that the service list necessarily adheres to any validity rules unless additional code interacting with Consul changes is checking for well formed-ness and validity) and that any change operation is ACID (but only with regard to Consul and the service list - this does not mean the transaction would incorporate additional needs such as the API Gateway updating new routes).
+  - When using the local configuration, some of the same issues exist as listed in the first alternative (validation of the file, etc.).
+  - Services would have to be notified by Consul if the service list changes or monitor the local config for a service list change in order to be able to address the add/removal of services at runtime.
+  - Again, how would the security bootstrapping address the add/removal of services and get new tokens established?  The bootstrapping service isn’t even long running to do this.
+  - We could use Consul for dynamic service needs and when Consul is not used resolve that the service list is static and requires a restart of the services.
+  - There is a mutually inclusive event issue (aka chicken and egg problem) with this solution in that we have to secure Consul via the security bootstrapping providing it a token and a list of all the services, but Consul is the place where the security service would get the list of all services.  The circumstances prohibit both to be true.
+- Use environment variables to provide the list of services.
+  - Each micro service would use the same env var to get the list of services.
+  - Use of an env var is a bit cumbersome in setup of all services – especially when the env var changes.  This is considered “unmanageable”.
+  - Adding or removing a service at runtime get tricky under this alternative (you can’t change the env var after startup)
+  - This would be a single, authoritative location for service list information.
+  - ACID needs would be moot as the env var would not be changed post startup.
+
+## Decision
+
+To be determined as part of a TSC and monthly architect’s meeting.
+
+## Consequences
+
+A key factor in determining a solution is whether EdgeX will support dynamic addition of services.  If the list of services is static and requires a restart of all of EdgeX when the list changes, the solutions are many and implementation straightforward.
+
+Adopters / users are requesting the ability to add/remove services at runtime (and operate in secure mode).
+
+## References
```

Comment:
Per request from @tonyespy, I will add a reference to the registry ADR in this section on the next cut.  https://docs.edgexfoundry.org/2.0/design/adr/0018-Service-Registry/